src/loggerMiddleware.lua

```diff
@@ -7,12 +7,12 @@ local function prettyPrint(value, indentLevel)
 	if typeof(value) == "table" then
 		table.insert(output, "{\n")
 
-		for key, value in pairs(value) do
+		for key, value_ in pairs(value) do
```

Comment:
I know we're fixing these to avoid shadowing, but we should probably also give them a name that means anything, like `subTableValue` or something.